binn/explainer.py

```diff
@@ -98,21 +101,49 @@ def explain_average(self,
             df = self.explain(test_data, background_data)
             dfs[iteration] = df
 
-        col_names = [f'value_{n}' for n in range(len(list(dfs.keys())))]
+        col_names = [f"value_{n}" for n in range(len(list(dfs.keys())))]
         values = [df.value.values for df in dfs.values()]
         values = np.array(values)
         values_mean = np.mean(values, axis=0)
         values_std = np.std(values, axis=0)
         df = dfs[0].copy()
-        df.drop(columns=['value'], inplace=True)
+        df.drop(columns=["value"], inplace=True)
         df[col_names] = values.T
-        df['value_mean'] = values_mean
-        df['values_std'] = values_std
-        df['value'] = values_mean
+        df["value_mean"] = values_mean
+        df["values_std"] = values_std
+        df["value"] = values_mean
         return df
 
+    def recursive_pathway_elimination(
```

Comment:
I'm not sure this should be integrated directly with BINNExplainer. It's additional functionality, so it might be good to keep it separate in it's own class, and we show the usage using documentation. that would also make it more flexible how we can interact with the data

Reply:
Sure, I actually had it like that before and was uncertain as to what is best. I don't mind either. I'll get it out to a separate module.